const.go

```diff
@@ -48,3 +52,10 @@ var NotaryDefaultExpiries = map[string]time.Duration{
 	"snapshot":  NotarySnapshotExpiry,
 	"timestamp": NotaryTimestampExpiry,
 }
+
+// NotaryDefaultHashes is the construct used to configure the default
+// supported hash algorithms.
+var NotaryDefaultHashes = map[string]string{
```

Comment:
Since this only seems to be used to get a list of the supported hashes, should we change this to an array?  That way, instead calling `NewFileMeta(bytes.NewReader(rootJSON), "sha256", "sha512")`, we can do something like `NewFileMeta(bytes.NewReader(rootJSON), notary.SupportedHashFunctions...)`?